assets/css/src/generic/_typography.scss

```diff
@@ -15,3 +15,25 @@ a {
     text-decoration: none;
   }
 }
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  color: $color-text;
+  font-weight: bold;
+  line-height: 1.25;
+  margin: 0 0 20px 0;
+}
+
+h4,
+h5 {
+  margin: 0 0 10px 0;
+}
+
+h1 { font-size: 32px; }
```

Comment:
Don't we need to scale down the heading sizes on smaller devices?

Reply:
No, the sizes stay the same